python/GafferUI/LabelPlugValueWidget.py

```diff
@@ -186,12 +193,10 @@ def __updateDoubleClickConnection( self ) :
 	def __labelDoubleClicked( self, label, event ) :
 
 		assert( label is self.__label )
-
-		if Gaffer.MetadataAlgo.readOnly( self.getPlug() ) :
-			return
+		assert( len( self.getPlugs() ) == 1 )
 
 		if self.__editableLabel is None :
-			self.__editableLabel = GafferUI.NameWidget( self.getPlug() )
+			self.__editableLabel = GafferUI.NameWidget( next( iter( self.getPlugs() ) ) )
 			self.__editableLabel._qtWidget().setMinimumSize( self.label()._qtWidget().minimumSize() )
```

Comment:
I think this is another spot where the old code is still sufficient.

Reply:
Reverted [here](https://github.com/GafferHQ/gaffer/pull/3980/commits/bfef18130f35960ab9096b0e92e7b6ee9ab854bc).